tsdb/engine/tsm1/engine.go

```diff
@@ -368,7 +369,7 @@ func (e *Engine) WritePoints(points []models.Point) error {
 }
 
 // DeleteSeries deletes the series from the engine.
```

Comment:
Update the comment to reflect the time range, "DeleteSeries removes points from the series whose times fall between min and max. The entire series is removed if all points fall within the specified range."